src/frontend/src/components/common/Navbar/navPages.ts

```diff
@@ -1,6 +1,6 @@
 export const NavPages = [
   {
-    id: 1,
+    id: 2,
```

Comment:
What is the significance of id: 2? I would try to avoid constants and have a more clear description what this is for.

Reply:
change back